docker/dockerfiles/polkadot/polkadot_builder.Dockerfile

```diff
@@ -19,7 +19,8 @@ LABEL description="Multistage Docker image for Polkadot: a platform for web3" \
 
 COPY --from=builder /polkadot/target/release/polkadot /usr/local/bin
 
-RUN useradd -m -u 1000 -U -s /bin/sh -d /polkadot polkadot && \
+USER root
```

Comment:
Changed back to an unprivileged user below, need root to execute those commands.